src/main/java/org/gbif/ipt/action/manage/AutoPublishAction.java

```diff
@@ -173,7 +179,22 @@ private void populateDays() {
   private void populateDaysOfWeek() {
     daysOfWeek = new LinkedHashMap<>();
     for (DayEnum dayOfWeek : DayEnum.values()) {
-      daysOfWeek.put(dayOfWeek.getIdentifier(), getText("manage.autopublish." + dayOfWeek.getIdentifier()));
+      daysOfWeek.put(
+          dayOfWeek.getIdentifier(), getText("manage.autopublish." + dayOfWeek.getIdentifier()));
+    }
+  }
+
+  private void setServerTimeZone() {
+    req.setAttribute("serverTimeZone", TimeZone.getDefault().getDisplayName(false, TimeZone.SHORT));
+  }
+
+  private void setUpdateFrequencyTime() {
+    // Retrieve the saved time and set it as a request attribute
+    if (resource.getPublicationMode().equals(PublicationMode.AUTO_PUBLISH_ON)) {
+      int savedHour = resource.getUpdateFrequencyHour();
+      int savedMinute = resource.getUpdateFrequencyMinute();
```

Comment:
It seems this still might cause an exception, see this method `isDeprecatedAutoPublishingConfiguration`. There might be some resources with old configuration without minutes (I managed to find a couple at the VertNet IPT actually, https://ipt.vertnet.org/manage/resource?r=chas_mammals). Make sure those values are not nulls, otherwise set them to 0